niyobertin/Express-google-auth
# Ft email on update

## What does the PR do?
- Sends the Email to the signed in User when there update on Profile
- The Email is sent using NodeMailer Package.
- Updating Email is restricted .
- The Email sent contains the updated fields on the profile
## How to Test this functionality?
- sign In using google 
- try to update fields on your Profile other than email
- check your InBox  on Gmail 
- you will see the email from "mugabo.kefa00" as our email handler User
-  and You are done 😎😎😎
## Packages used and useful links
- nodemailer from https://www.nodemailer.com/
- passport  from https://www.passportjs.org/
